diff -Naru apache-log4cxx-0.10.0.orig/src/examples/cpp/console.cpp apache-log4cxx-0.10.0/src/examples/cpp/console.cpp
--- apache-log4cxx-0.10.0.orig/src/examples/cpp/console.cpp	2008-04-01 07:34:52.000000000 +0900
+++ apache-log4cxx-0.10.0/src/examples/cpp/console.cpp	2014-06-10 19:46:39.264105002 +0900
@@ -16,6 +16,8 @@
  */
 
 #include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
 #include <log4cxx/logger.h>
 #include <log4cxx/consoleappender.h>
 #include <log4cxx/simplelayout.h>
diff -Naru apache-log4cxx-0.10.0.orig/src/main/cpp/inputstreamreader.cpp apache-log4cxx-0.10.0/src/main/cpp/inputstreamreader.cpp
--- apache-log4cxx-0.10.0.orig/src/main/cpp/inputstreamreader.cpp	2008-04-01 07:34:09.000000000 +0900
+++ apache-log4cxx-0.10.0/src/main/cpp/inputstreamreader.cpp	2014-06-10 19:46:37.960104949 +0900
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+#include <string.h>
 #include <log4cxx/logstring.h>
 #include <log4cxx/helpers/inputstreamreader.h>
 #include <log4cxx/helpers/exception.h>
diff -Naru apache-log4cxx-0.10.0.orig/src/main/cpp/socketoutputstream.cpp apache-log4cxx-0.10.0/src/main/cpp/socketoutputstream.cpp
--- apache-log4cxx-0.10.0.orig/src/main/cpp/socketoutputstream.cpp	2008-04-01 07:34:09.000000000 +0900
+++ apache-log4cxx-0.10.0/src/main/cpp/socketoutputstream.cpp	2014-06-10 19:46:37.980105045 +0900
@@ -15,6 +15,7 @@
  * limitations under the License.
  */
 
+#include <string.h>
 #include <log4cxx/logstring.h>
 #include <log4cxx/helpers/socketoutputstream.h>
 #include <log4cxx/helpers/socket.h>
